action.yml

```diff
@@ -26,6 +26,10 @@ inputs:
     type: boolean
     default: false
     description: "Disable caching of bun executable."
+  token:
+    required: false
+    default: ${{ github.token }}
+    description: "Personal access token (PAT) used to fetch tags from oven-sh/bun repository"
```

Comment:
It's more to access api.github.com with the per token rate limit instead of the per IP one. The documentation should mention that is recommended for user using wildcard or range version.